packages/devextreme/js/__internal/events/pointer/m_observer.ts

```diff
@@ -1,69 +1,67 @@
-import { each } from '../../core/utils/iterator';
-import readyCallbacks from '../../core/utils/ready_callbacks';
-import domAdapter from '../../core/dom_adapter';
-
-const addEventsListener = function(events, handler) {
-    readyCallbacks.add(function() {
-        events
-            .split(' ')
-            .forEach(function(event) {
-                domAdapter.listen(domAdapter.getDocument(), event, handler, true);
-            });
-    });
+import domAdapter from '@js/core/dom_adapter';
+import { each } from '@js/core/utils/iterator';
+import readyCallbacks from '@js/core/utils/ready_callbacks';
+
+const addEventsListener = function (events, handler) {
+  readyCallbacks.add(() => {
+    events
+      .split(' ')
+      .forEach((event) => {
+        (domAdapter as any).listen(domAdapter.getDocument(), event, handler, true);
```

Comment:
The same https://github.com/DevExpress/DevExtreme/pull/27779/commits/5550d8588b4ae20f64a01e6c5e07441709e5ebef?diff=unified&w=1#r1695242659